src/App.js

```diff
@@ -173,6 +189,9 @@ class App extends React.Component {
             govActPrevActionHash: undefined,
             govActPrevActionIndex: undefined,
             proposalPolicy: undefined,
```

Comment:
```suggestion
            suppliedGuardrailScriptHash: undefined,
```
I would also rename this variable throughout, for clarity

Reply:
I will do this in another PR